### DISCUSSION TITLE
Finetune with large energy shift

### DISCUSSION THREAD

**Discussion by chtchelkatchev** [deepmodeling/deepmd-kit#3311] [General]
When I fine tune a pretrained DeepMD model by a datatset with large shifts in elemental energy problems may arise. For example, Li has -0.95 eV/atom in GGA (https://next-gen.materialsproject.org/materials/mp-135/tasks/mp-990455) and -1.17 eV/atom in R2SCAN (https://next-gen.materialsproject.org/materials/mp-135/tasks/mp-1943895). How I should modify finetuning procedure? Can you provide an example with docs? What magnitude of the energy shift is crititcal for ``standard'' finetuning described in deepmd docs (https://docs.deepmodeling.com/projects/deepmd/en/master/train/finetuning.html)? I know about "atom_ener" parameter... There is a detailed discussion of the finetuning problem in the CHGnet documentation (https://github.com/CederGroupHub/chgnet/blob/main/examples/fine_tuning.ipynb)...

**Comment by njzjz** [comment]
Hi, I hope I've got what you mean. The bias here may not be equal to the elemental energy (the energy of a single atom without neighbors), as the NN will contribute another bias. The fine-tune procedure only ensures that the total energy predicted by the model fits that in your dataset.

**Reply by chtchelkatchev** [reply]
Thanks for the answer. The question is most likely related to the lack of understanding of how atom_ener works. This option seems to help if the finetuning dataset corresponds to energies of isolated atoms that are very different from those for the training dataset. Do I understand correctly that atom_ener simply automatically subtracts the energies of isolated atoms from the energy of the atomic configuration of the dataset (or is it something more complex)? Does this option work with finetuning? If this option was not used when training the neural network, can this option be used during finetuning?
